src/encoded/static/components/cart/menu.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * Components to display the status of the cart in the navigation bar, and to navigate to cart
+ * pages.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { DropdownMenu, DropdownMenuSep } from '../../libs/ui/dropdown-menu';
+import { NavItem } from '../../libs/ui/navbar';
+import { svgIcon } from '../../libs/svg-icons';
+import { truncateString } from '../globals';
+import Status from '../status';
+import { CartClearModal } from './clear';
+import CartShare from './share';
+
+
+/**
+ * Renders the cart icon menu and count or in-progress spinner in the nav bar.
+ */
+const CartNavTitle = ({ elements, inProgress, locked }) => {
+    let status;
```

Comment:
I originally called the little icon a cart “status” which now is confusing since the cart’s `status` property has new meaning. `status === undefined` means no little icon to display.